h/accounts/schemas.py

```diff
@@ -85,6 +86,13 @@ def unblacklisted_username(node, value, blacklist=None):
         raise colander.Invalid(node, msg)
 
 
+def required_acceptance(node, value):
+    '''Colander validator that ensures privacy acceptance checkbox checked'''
+    if value != 'true':
```

Comment:
Yes, literal string value of `true`. Long story as to my struggles with actual `True`.

Reply:
Definitely curious to hear your tail of woe.